.teamcity/src/main/kotlin/configurations/SmokeTests.kt

```diff
@@ -20,6 +21,11 @@ class SmokeTests(model: CIBuildModel, stage: Stage, testJava: JvmCategory, task:
         }
     }
 
+    requirements {
+        // Smoke tests is usually heavy and the build time is twice on EC2 agents
```

Comment:
I'm a bit hesitant if we should run smoke tests on EC2. I suggest we try to disable it on ec2 and see what's the impact on overall feedback time for PRF stage.